Sming/compiler/ld/eagle.app.v6.cpp.include.ld

```diff
@@ -0,0 +1,217 @@
+/*  
```

Comment:
Please, rename this file `eagle.app.v6.cpp.include.ld` to  `common.ld`.  And that one `eagle.app.v6.cpp.ld` to standalone.rom.ld. And apply the changes to the make files. IMHO it makes it easier to figure out what are the different LD script for.